packages/d3fc-webgl/src/series/glPoint.js

```diff
@@ -19,9 +19,9 @@ export default () => {
 
     program
         .buffers()
-        .attribute('aXValue', xValueAttribute)
-        .attribute('aYValue', yValueAttribute)
-        .attribute('aSize', sizeAttribute);
+        .attribute('aCrossValue', xValueAttribute)
+        .attribute('aMainValue', yValueAttribute)
+        .attribute('aArea', sizeAttribute);
```

Comment:
Please can you make this `aSize` and then I think we're good

Reply:
Done.